Community-Supported/defragment-hyper-file/README.md

```diff
@@ -0,0 +1,69 @@
+# defragment-data-of-existing-hyper-file
+## __defragment_data_of_existing_hyper_file__
+
+
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+__Current Version__: 1.0
+
+This sample demonstrates how you can optimize the file storage of an existing `.hyper` file by copying all of the tables and data into a new file, in a continuous sequence, to eliminate any fragmentation that might have occurred. This method can improve the performance of your Hyper file when you have large amounts of data, as fragmentation can increase both file size and access times.
+
+This sample should serve as a starting point for anyone looking for a programmatic way to reduce the fragmentation of their `.hyper` file.
+
+For a description of how fragmentation can occur in the `.hyper` file and for ways of minimizing it's occurrence, see [Optimize Hyper File Storage](https://help.tableau.com/current/api/hyper_api/en-us/docs/hyper_api_defrag.html).
+
+# Get started
+
+## __Prerequisites__
+
+To run the script, you will need:
+
+- a computer running Windows or macOS
```

Comment:
Linux?

Reply:
maybe just reference the requirements from the HyperAPI docs